ddtrace/contrib/futures/threading.py

```diff
@@ -21,10 +21,12 @@ def _wrap_submit(func, instance, args, kwargs):
     if ddtrace.tracer.context_provider._has_active_context():
         current_ctx = ddtrace.tracer.context_provider.active()
 
-    # extract the target function that must be executed in
-    # a new thread and the `target` arguments
-    fn = args[0]
-    fn_args = args[1:]
+    # The target function can be provided as a kwarg argument "fn" or the first positional argument
```

Comment:
I was going to suggest using `get_argument_value` but we do unconventional things with the positional args so this is :+1: by me